Expression-atlas-query-RShiny/README.md

```diff
@@ -0,0 +1,54 @@
+- Prerequisite libraries:
+  - R packages: 
+    `intall.packages(c("shiny", "ggbeeswarm", "plyr", "gridExtra", "tidyverse", "httr", "RJSONIO"))`
+  - Ask us for the ODM R client libraries, and then 
+    `install.packages(c("studyCurator", "integrationCurator"))`
+
+Run the Shiny application in the R console (R/RStudio/Jupyter notebooks)
+- Copy and paste your API token to the "token" variable in scripts app.R and dictionaries_api.R (e.g. `token <- "<your token here>""`), and save the files
+- Set your working directory to where the Shiny app folder locates using `setwd('~/shinyappfolder')`
+  load Shiny package using `library(shiny)`
+- Run the app using `runApp()`
+
+Alternatively, run the Shiny application in a terminal or console window
+- Copy and paste your API token to the "token" variable in both app.R and dictionaries_api.R (e.g. `token <- “<your token here>""`), and save the files
+- Run the app using `R -e “shiny::runApp('~/shinyappfolder')"`
+
+**Example queries:**
+- Explore bulk RNA-Seq study:
+  - Once you successfully load the Shiny app, a web-page style user interface will appear.
+  On the left panel, there are multiple text boxes and drop-down lists that you can use to filter data for your analysis.
+  - Select a type of study from the "Study Type", e.g. **Bulk Study**.
+  - Select an area of interest in the "Therapeutic area" list, e.g. **Immunology**. You can either
+    - Type in the term, and an automated matched list will be returned for you to choose from, OR
+    - Directly select from the drop-down list.
+  - Select a study by ticking one or more studies appeared underneath the "Therapeutic area", e.g. ticking the study **GSF1061557 (Reyes et al.)**.
+  - Choose a gene of interest and type into the "Genes", e.g. **CD3E**.
+  - You will be able to see both transcriptomic and proteomic data shown in the main panel under the "Beeswarm Plot" tab. You can also view relevant information in other tabs.
+  - You can further filter the data using the "Cell types", e.g. typing in **lymphocyte**. Can you spot the difference? There are less cell types shown in the "Beeswarm Plot" and "Sample Info" tabs responding to the filter.
+  - You can further filter the data using "Sample filter" (e.g. **Sex=female**) and "Expression filter" (e.g. **Source="RNA-Seq"**). All information in the "Sample Info" and "Expression Info" can be used to slice the data.
+  - You can also query two genes at the same time, e.g. typing in **CD28** next to CD3E. Two plots are displayed for the chosen two genes.
+
+[Note]: If you have also select GSF1061820 (EU BLUEPRINT) study, a stacked cell types will be shown without filtering “Cell Types” as lots of cell types available in this study.
```

Comment:
Maybe phrase this bit to this: "Note: the EU Blueprint study has more granular cell types compared to the Reyes et al. study. To display expression data from both studies together, add 'lymphocyte' cell type filter, which will filter, merge, and show the cell types from both studies at the same level of granularity.